R/config.R

```diff
@@ -132,6 +132,8 @@ Data <- S7::new_class(
 #' Formatted as "YYYY-MM-DD".
 #' @param disease A string specifying the disease being modeled. One of
 #'    `"COVID-19"` or `"Influenza"` or `"RSV"`.
+#' @param low_case_count_thresholds A list of thresholds to use for
```

Comment:
Can you add what the threshold values should be between? I'm guessing between 0 and 1?